src/main/java/org/example/page/vietjet/HomePage.java

```diff
@@ -0,0 +1,145 @@
+                selectNumberOfChildren(passenger.getChildren());
+            }
+            if (Objects.nonNull(passenger.getInfants())) {
+                selectNumberOfInfants(passenger.getInfants());
+            }
+            getTextboxByLabel(Label.PASSENGER).click();
+        }
+    }
+
+    public void searchTicket(SearchTicketData searchTicketData) {
+        fillTicketInfo(searchTicketData);
+        clickSearchButton();
+    }
+
+    @Step("Search and select the first result: {0} {1}")
+    public void searchAndSelectTheFirstResult(Label searchField, String searchKeyword) {
+        getTextboxByLabel(searchField).setValue(searchKeyword).click();
+        placeOptions.first().shouldBe(Condition.visible).scrollIntoView(false).click();
+    }
+
+    @Step("Select data: {0}")
+    public void selectDatePicker(LocalDate localDate) {
+        String country = (String) YamlUtils.getProperty("config.country");
+        String language = (String) YamlUtils.getProperty("config.language");
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMMM yyyy", new Locale.Builder()
+                .setLanguage(language)
+                .setRegion(country)
+                .build());
+        getDate(localDate, formatter).click();
+    }
+
+    @Step("Select number of adults: {0}")
+    public void selectNumberOfAdults(int targetNumber) {
+        int currentNumber = Integer.parseInt(adultsValue.getText());
+        SelenideElement buttonToClick = currentNumber < targetNumber ? plusAdultsButton : minusAdultsButton;
+        for (int i = 0; i < Math.abs(targetNumber - currentNumber); i++) {
+            buttonToClick.click();
+        }
+    }
+
+    @Step("Select number of adults: {0}")
+    public void selectNumberOfChildren(int targetNumber) {
+        int currentNumber = Integer.parseInt(childrenValue.getText());
+        SelenideElement buttonToClick = currentNumber < targetNumber ? plusChildrenButton : minusChildrenButton;
+        for (int i = 0; i < Math.abs(targetNumber - currentNumber); i++) {
+            buttonToClick.click();
+        }
+    }
+
+    @Step("Select number of adults: {0}")
+    public void selectNumberOfInfants(int targetNumber) {
+        int currentNumber = Integer.parseInt(babyValue.getText());
+        SelenideElement buttonToClick = currentNumber < targetNumber ? plusBabyButton : minusBabyButton;
+        for (int i = 0; i < Math.abs(targetNumber - currentNumber); i++) {
+            buttonToClick.click();
+        }
+    }
+
+    @Step("Click on Search button")
+    public void clickSearchButton() {
+        getButtonByText((String) YamlUtils.getProperty("button.letgo")).click();
+    }
+
+    private SelenideElement getTextboxByLabel(Label label) {
+        return $x(String.format("//div[label[.='%s']]//input", label.toString()))
+                .shouldBe(Condition.visible);
+    }
+
+    private SelenideElement getDate(LocalDate localDate, DateTimeFormatter formatter) {
```

Comment:
Is this method used to click on a date on the calendar, isn't it? You should get the formatter inside this method rather than passed via argument!